packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -321,12 +321,6 @@ def test_dcos_diagnostics_report(dcos_api_session):
         assert len(report_response['Nodes']) > 0
 
 
-@pytest.mark.supportedwindows
-@pytest.mark.xfailflake(
-    jira='DCOS-44935',
-    reason='test_dcos_diagnostics.test_dcos_diagnostics_bundle_create_download_delete is Flaky',
-    since='2018-11-20',
-)
```

Comment:
Can you please also update test body so it looks similar to master?

Reply:
What do you mean? Changing the lists of files? Because I'd rather not do that in this PR since it would mean changes to the config and the test assertions.